TWLight/templates/staff.html

```diff
@@ -0,0 +1,62 @@
+{% extends "base.html" %}

```

Comment:
This seems like the kind of thing that we should be using the delivered admin tools for instead of writing our own interface. It is possible to give staff limited access to the admin site.  Are we needing something more granular than what can be done there? I think we've talked about parts of this before, but it would be worth revisiting.

Reply:
Perhaps, though isn't this is a little complex for the admin dashboard, at least in it's current form? This parses data from a CSV to turn into objects, could we do that in the admin interface?

Additionally, I approached this from the basis that a staff dashboard for doing things like this was something we were discussing (i.e. this page could be expanded to include adding a new partner, assigning a coordinator, etc.). I appreciate that one of the other options we talked about was limited /admin access though.

Reply:
Sorry for being slow to respond to this question. To answer:
* Yes, I think this functionality can be added in the admin dashboard.
* There is nothing wrong with this approach, but it implements a design decision that I wasn't aware we had reached. There be consequences.

Reply:
Note to self, something like: https://stackoverflow.com/questions/3974620/import-csv-data-into-database-in-django-admin